src/assets/css/main.css

```diff
@@ -431,3 +435,15 @@ br.big {
 .skip-link:focus {
   top: 0;
 }
+
+.comment,
+.markdown-textarea {
+  max-width: 50rem;
+}
+
+input[type="checkbox"],
+.form-check label,
+select,
+option {
+  cursor: pointer;
```

Comment:
This feels like it doesn't belong in this PR; plus I'm not sure pointers for these things is correct.

Reply:
Should selects not have the cursor be a pointer? The label? both?

Reply:
I'm kind of a stickler in that I think the default cursor styles for form elements is handled correctly by browsers by default -- a standard `default` arrow cursor, like it is on every other form element in your operating system. The `pointer` style was designed for hyperlinks.

I know this convention has changed in recent years but I'm not sure it's necessarily the "right" thing to do.